deploy/complete/helm-chart/mushop/charts/carts/templates/oadb-secrets.yaml

```diff
@@ -1,7 +1,7 @@
 {{- $usesOsb := (index (.Values.global | default .Values) "osb").atp | default .Values.osb.atp -}}
 {{- $oadbService := .Values.secrets.oadbService | default .Values.global.oadbService -}}
 {{- $oadbWalletPw := .Values.secrets.oadbWalletPassword | default .Values.global.oadbWalletPassword -}}
-{{- $oadbUser := .Values.secrets.oadbUser | default (printf "%s_%s_user" .Release.Namespace .Chart.Name ) | trunc 96 -}}
+{{- $oadbUser := .Values.secrets.oadbUser | default (printf "%s_%s_user" .Release.Namespace .Chart.Name ) | replace "-" "_" | trunc 96 -}}
```

Comment:
replacing affects any other place? the user will not be confused that tried something like mushop-something and end up with mushop_something?

Reply:
Nah, usually we don’t even go to that level of depth